node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -44,6 +44,10 @@ object Configuration {
   private val DefaultKeyFileName                = "node.key.pem"
   private val DefaultSecureRandomNonBlocking    = false
   private val DefaultMaxNumOfConnections        = 500
+  private val DefaultRequiredSigns              = 0
```

Comment:
I've no idea what this property does, but 0 for number of required signatures seems dangerous :)

Reply:
Well, true. @birchmd what do you propose? We need some default value. Make it very big?

Reply:
The default is for convenience in testing mostly. In production there should be no default so that anyone launching a private network has to make that decision for themselves.  Maybe make a ticket to remove the default value some time in November.